### PR TITLE
Add curl to sysdeps.

### LINKED ISSUE #3295: sysdeps missing curl
`sysdeps` first step is to use `curl` to download the script to install node but `curl` isn't installed by default.

https://github.com/juju/juju-gui/blob/develop/Makefile#L83

### PR DESCRIPTION
**Pull request by hatched**
Fixes #3295

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8234/

File: Makefile [hunk 1]
```diff
@@ -80,6 +80,7 @@ help:
 #########
 .PHONY: sysdeps
 sysdeps:
+	sudo apt install curl
```
Comment (frankban) [inline Makefile]: Add -y?

Comment (hatched) [reply Makefile]: Good call, added.

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Build finished.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8237/

**Pull request merged**